frontend/src/components/chip/chip.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from '@emotion/styled'
+import {HTMLAttributes, PropsWithChildren} from 'react'
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  large?: boolean
+  selected: boolean
+  enabled: boolean
+}
+
+export function Chip({large, selected, enabled, children, ...props}: PropsWithChildren<Props>) {
```

Comment:
[default value](https://arc.net/l/quote/ootpruvs) 을 적극 사용해보면 좋을거 같아요!
`undefined` 값이 넘어올 때 기본값 설정하기 유용한 방식이에요

```suggestion
export function Chip({size = 'sm', selected, enabled, children, ...props}: PropsWithChildren<Props>) {
```